src/components/ContactUs/DesktopVersion.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export default function DesktopVersion() {
+  return (
+    <>
+      <h1 className="hidden lg:block font-montserrat text-[32px] font-[900] w-fit mx-auto py-4">
+        Have an interesting idea we should discuss?
+      </h1>
+      <div className="w-3/4 lg:bg-[#CEEDF4BF] mx-auto pt-[74px] pb-[46px] rounded-[10px]">
+        <form className="mx-auto w-2/4">
+          <label
+            htmlFor="name"
+            className="block mb-2 text-xs text-black/[0.44] font-dmSans"
+          >
+            Name
+          </label>
+          <input
+            id="name"
+            type={'text'}
+            placeholder={'Jane Doe'}
+            className="border border-[#CEEDF4] w-full rounded-lg px-4 py-3 mb-5 placeholder-[#7E7979] outline-none font-montserrat text-base"
+            required
+          />
+          <label
+            htmlFor="email"
+            className="block mb-2 text-xs text-black/[0.44] font-dmSans"
+          >
+            Email
+          </label>
+          <input
+            id="email"
+            type={'you@example.com'}
```

Comment:
This sould be `type="email"`.

Reply:
Fixed via [ad23c13](https://github.com/reactdeveloperske/reactdevske-website/pull/102/commits/ad23c1319197d6098b03d8e4fdf6439d82f2f4ce)